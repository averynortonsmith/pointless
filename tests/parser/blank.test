
(a, _, c) = foo
(_, _) = 123
