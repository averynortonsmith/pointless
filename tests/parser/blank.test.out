(Program 
 null
 []
 (Def 
  (Tuple 
   (Name "a")
   (Blank)
   (Name "c"))
  (Name "foo"))
 (Def 
  (Tuple 
   (Blank)
   (Blank))
  (Number 123)))
